login/login-tool/tool/src/webapp/WEB-INF/xlogin-context.saml.adfs-prod.xml

```diff
@@ -278,6 +278,7 @@
     </bean>
 
     <!-- Initialization of OpenSAML library -->
+    <!-- Use SHA256SAMLBootstrap for SHA-256 instead of default SHA-1 -->
     <bean class="org.springframework.security.saml.SAMLBootstrap"/>
```

Comment:
Doesn't this bean class name need to change then, to org.sakaiproject.login.springframework.SHA256SAMLBootstrap ?

Reply:
Only if you want to use the custom SHA-256 bootstrap, yes

Reply:
May be clearer to a second commented-out bean definition? e.g.

````
    <!-- Initialization of OpenSAML library -->
    <bean class="org.springframework.security.saml.SAMLBootstrap"/>

    <!-- Use SHA256SAMLBootstrap for SHA-256 instead of default SHA-1 -->
    <!-- bean class="org.sakaiproject.login.springframework.SHA256SAMLBootstrap"/ -->
````

Reply:
okay done